app-core/src/main/java/net/spals/appbuilder/app/core/generic/GenericWorkerApp.java

```diff
@@ -34,13 +39,21 @@
 
     public static class Builder extends GenericWorkerApp_Builder implements WorkerAppBuilder<GenericWorkerApp> {
 
+        private final List<Module> orderedModules = new ArrayList<Module>();
         private final LifecycleInjectorBuilder lifecycleInjectorBuilder;
         private final ServiceGraph serviceGraph = new ServiceGraph();
```

Comment:
Question, is this part of freeBuilder setting the variables in the declaration vs in the constructor?